plugin/napari_lattice/_reader.py

```diff
@@ -138,3 +141,24 @@ def bdv_h5_reader(path):
 
     layer_type = "image"  # optional, default is "image"
     return [(images, add_kwargs, layer_type)]
+
+
+def tiff_reader(path):
+    """Take path to tiff image and returns a list of LayerData tuples.
+    Specifying tiff_reader to have better control over tifffile related errors when using AICSImage
+    """
+    
+    try:
+        image = AICSImage(path)
+    except Exception:
+        import sys
+        exception = sys.exc_info()[0]
+        print("Error Reading Tiff: ", sys.exc_info()[
+                0], "has occurred. Try upgrading tifffile library: pip install tifffile --upgrade.")
+        raise

```

Comment:
```suggestion
    except Exception as e:
        raise Exception("Error reading TIFF. Try upgrading tifffile library: pip install tifffile --upgrade.") from e
```
This is exception chaining: https://docs.python.org/3/tutorial/errors.html#exception-chaining

Reply:
thats neat!